config/routes.rb

```diff
@@ -5,9 +5,15 @@
   get '/auth/:provider/callback', to: 'sessions#create'
   get '/logout', as: :logout, to: 'sessions#destroy'
 
+  get '/profile', to: 'users#show'
+
   get 'static_place', to: 'static#place'
   get '/autocomplete', to: 'google/autocomplete#index'
 
   get "places/:slug", as: 'place', to: 'places#show'
   resources 'places', only: [:create]
+
+  post '/follow', to: 'user_follows#create'
+  get '/follow', to: 'user_follows#show'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.